joplin/api/schema.py

```diff
@@ -231,6 +231,22 @@ class Meta:
         interfaces = [graphene.Node]
 
 
+class EventFilter(FilterSet):
```

Comment:
Can we see an example of where this is used? 
This is probs a stretch goal (and maybe we can make a separate issue), but we should have a test that this ordering filter works correctly if we're going to  roll our own schema ( or we can use https://github.com/linuxsoftware/ls.joyous which already has test coverage )

Reply:
example of where it's used on Janis? getAllEvents in static.config

Reply:
here's an example of a simple API test case btw: 
https://github.com/cityofaustin/joplin/pull/545/files